site/guide/model-documentation/_view-documentation-activity-overview.qmd

```diff
@@ -0,0 +1,21 @@
+ The **Model Activity** page shows a record of all activities, including actions performed by users in your organization and updates generated via the {{< var vm.developer >}} such as:
+
+:::: {.content-visible unless-format="revealjs"} 
+- Inventory fields updated on the model^[[Edit model inventory fields](/guide/model-inventory/edit-model-inventory-fields.qmd)]
+- Model status transitions^[[Working with model workflows](/guide/model-workflows/working-with-model-workflows.qmd)]
+- Updates to model documentation, validation reports,^[[Working with model documentation](working-with-model-documentation.qmd)] or ongoing monitoring reports^[[Ongoing monitoring](/guide/monitoring/ongoing-monitoring.qmd)]
+- Test results added to your model^[[ValidMind Developer Framework](/developer/get-started-developer-framework.qmd)]
+- Findings added, updated, or removed^[[Work with model findings](/guide/model-validation/work-with-model-findings.qmd)]
+- Comment creation and replies^[[Collaborate with others (Commenting)](/guide/model-documentation/collaborate-with-others.qmd#commenting)]
+
+::::
+
+:::: {.content-hidden unless-format="revealjs"}
```

Comment:
A question for my own understanding of our convention: We have https://docs-demo.vm.validmind.ai/pr_previews/beck/sc-5949/documentation-adding-filters-to-the-model/about/contributing/style-guide/conventions.html#margin-footnotes but it doesn't specify how links in instructional content should be done, only to minimize them. 

IIRC, these margin footnotes do not work as-is in reveal.js, so do we just code them as regular Markdown links or ...?

Reply:
That's exactly right, they do not show up in the reveal-js at all. Which means that inline links is the only option for that version if we really want to link out to supplementary articles.

Reply:
It's mainly for my own edification but might be helpful. Opened https://app.shortcut.com/validmind/story/6326